nginx/opt/containerbuddy/nginx.json

```diff
@@ -16,5 +18,50 @@
       "poll": 7,
       "onChange": "/opt/containerbuddy/reload-nginx.sh"
     }
-  ]
+  ],
+  "telemetry": {
+    "name": "telemetry",
+    "url": "/telemetry",
```

Comment:
Strictly speaking this is a design question about Containerbuddy's telemetry implementation. So I'm moving this conversation to https://github.com/joyent/containerbuddy/pull/119#issuecomment-206856695 so that anyone following Containerbuddy can follow this discussion.